bin/IdamHelper.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const fetch = require('node-fetch');
+const { URLSearchParams } = require('url');
+const btoa = require('btoa');
+
+const USERNAME = 'docassemblytestuser@hmcts.net';
+const PASSWORD = '4590fgvhbfgbDdffm3lk4j';
+const FIRSTNAME = 'docassembly';
+const LASTNAME = 'testuser';
+
+const Env = require('./Env');
+
+class IdamHelper {
+
+  constructor() {
+    this.idamUrl = Env.getIdamUrl();
+    this.client = Env.getOAuthClient();
+    this.secret = Env.getOAuthSecret();
+    this.redirect = Env.getOAuthRedirect();
+  }
+
+  async getIdamToken() {
+    await this.createUser();
+    const code = await this.getCode();
+    console.log(`this is the code ${code}\n`);
+    const token = await this.getToken(code);
+    console.log(`this is the token ${token}\n`);
+    return `Bearer ${token}`;
+  }
+
+  createUser() {
```

Comment:
```
private async createUser(): Promise<void>
```

Reply:
that is not supported with current version of node/javascript